lib/i18n_routable/generating/localizable_url_helper.rb

```diff
@@ -4,26 +4,25 @@ module LocalizableUrlHelper
 
       # we override this, but it really only removes positional_keys that are
       # locale or i18n_*
-      def define_url_helper_with_localize(route, name, kind, options)
```

Comment:
I was playing around with this and removed this entire file and the tests still passed. All the extension was supposed to do is remove `:locale` and `:i18n_` segments, and it looks like `ActionDispatch::Routing::RouteSet::NamedRouteCollection::UrlHelper` might do that now already.